README.md

```diff
@@ -48,7 +48,7 @@ That's it.
 This plugin is compatible with:
 
 - PHP **5.4+**, **7.x**, and **8.x** (Support for PHP v8 is available since [`v0.7.0`][v0.7])
-- [Composer][composer] **1.x** and **2.x** (Support for Composer v2 is available since [`v0.7.0`][v0.7])
+- [Composer][composer] **2.x**
```

Comment:
Is there a reason to remove the historic context remark ?
Might it be more informative to keep it and add info about when support for Composer 1.x was dropped ?

Reply:
After I removed the "1.x and", the words in parenthesis no longer made sense. I considered rewriting this section into a table; would this be preferable?

Reply:
Perhaps a table like this:

|Plugin version|PHP|Composer|PHP_CodeSniffer|
|-|-|-|-|
|`^0.1.0`|`*`|`^1.0`|`*`|
|`^0.2.0`|`*`|`^1.0`|`*`|
|`^0.3.0`|`*`|`^1.0`|`*`|
|`^0.4.0`|`*`|`^1.0`|`*`|
|`^0.5.0`|`^5.3 \|\| ^7.0`|`^1.0`|`^2.0 \|\| ^3.0`|
|`^0.6.0`|`^5.3 \|\| ^7.0`|`^1.0`|`^2.0 \|\| ^3.0`|
|`^0.7.0`|`>=5.3`|`^1.0 \|\| ^2.0`|`^2.0 \|\| ^3.0`|
|`^1.0`|`>=5.4`|`^1.0 \|\| ^2.0`|`^2.0 \|\| ^3.1 \|\| ^4.0`|
|`^1.1`|`>=5.4`|`^2.0`|`^2.0 \|\| ^3.1 \|\| ^4.0`|

Reply:
I don't think a table is needed. That's for Composer to sort out on install. Just some generic info should be enough.

Reply:
> some generic info should be enough.

So, should we leave this as-is, or do you want to supply some different text?

Reply:
What about something like:
`(Support for Composer v2 is available since [`v0.7.0`][v0.7]; Support for Composer < 2.0. was dropped in [`v1.1.0`][v1.1])`

@Potherca What do you think ?